ethash/src/cache.rs

```diff
@@ -265,6 +266,7 @@ fn read_from_path(path: &Path) -> io::Result<Vec<Node>> {
 	}
 
 	let out: Vec<Node> = unsafe {
+		#[allow(clippy::cast_ptr_alignment)] // TODO: Why is this okay?
```

Comment:
hmm, this is a good question, I'm not sure myself about the alignment cc @Vurich